src/agent/agent_info/include/agent_info.hpp

```diff
@@ -19,6 +19,9 @@ class AgentInfo
     void SetUUID(const std::string& uuid);
     void SetGroups(const std::vector<std::string>& groupList);
 
+    std::string CreateKey();
```

Comment:
These two functions seem to be like utils, I don't know if this is the right place for them.

Reply:
Maybe they can be private and be called inside `SetKey`.

Reply:
Fixed!